LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Rhythmic Technologies, Inc.
+Copyright (c) 2020 Rhythmic Technologies, Inc.
```

Comment:
Should we be doing this for all our repos? I bet there's a GH Actions bot for that. It'd be a cool thing to set up